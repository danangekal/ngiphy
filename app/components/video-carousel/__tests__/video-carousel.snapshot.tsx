import { render } from '@testing-library/react';

import VideoCarousel from '../index';

it('VideoCarousel component unchanged', () => {
  const { container } = render(<VideoCarousel />);
  expect(container).toMatchSnapshot();
});
