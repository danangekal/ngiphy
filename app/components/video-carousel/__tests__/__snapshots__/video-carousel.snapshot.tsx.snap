// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoCarousel component unchanged 1`] = `
<div>
  <div
    class="flex justify-center items-center"
  >
    <div
      class="max-w-xl w-full relative"
    >
      <div
        class="overflow-hidden rounded-lg"
      >
        <div
          class="flex"
        />
      </div>
      <div
        class="absolute top-0 left-2 right-2 flex items-center justify-between text-4xl text-white hover:text-blue-200"
      >
        <button>
          &lt;
        </button>
        <button>
          &gt;
        </button>
      </div>
    </div>
  </div>
</div>
`;
